home-music/home-music.css

```diff
@@ -0,0 +1,171 @@
+* {
+  margin: 0;
+  padding: 0;
+  border-radius: 0;
+  scroll-behavior: smooth;
+}
+
+:root {
+  --main-color: #210c39;
+  --secondary-color: #ffffff;
+  --third-color: #6644b8;
+  --theme-switcher: #f6d003;
+}
+
+body {
+  font-family: "Alef", sans-serif;
+}
+
+/* Starting the styles for the mobile screens. */
+/* All styles in the mobile sections were done by taking the screen of IPhone 12 pro as design theme. */
+.container {
+  width: 100%;
+  height: 100vh;
+  background-image: linear-gradient(to bottom, #17082a, #301353, #17082a);
+  overflow: hidden;
+}
+
+.container .icons {
+  width: 95%;
+  padding: 0.5em;
+  margin-top: 0.5em;
+  display: flex;
+  justify-content: space-between;
+  color: var(--secondary-color);
+}
+
+.container .icons .landing-page .arrow,
+.container .icons .sun {
+  font-size: 1.2em;
+}
+
+.container .icons .landing-page {
+  color: var(--secondary-color);
+  text-decoration: none;
+}
+
+.container .icons .sun {
+  color: var(--theme-switcher);
+  cursor: pointer;
+}
+
+.container .content {
+  position: relative;
+}
+
+.container .content #music-pic {
+  position: absolute;
+  left: 4rem;
+  top: -8rem;
+}
+
+.container .content .rest-cont {
+  width: 100%;
+  position: absolute;
```

Comment:
The rest-cont div is wrongly placed in the page, you may need to change the position absolute.

Reply:
resolved